docs/examples/general/data_loading/webdataset.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    folder_dataset_files = glob.glob(os.path.join(root_path, "devel-[012]", "*.jpg"))
</pre><p>I think you could use a temp dir instead of keeping the untarred dirs in DALI_extra</p><pre class="ql-syntax" spellcheck="false">tmp_dir = tempfile.TemporaryDirectory()
# Untar to tmp_dir
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>